Voycar.Api.Web/Entities/User.cs

```diff
@@ -7,25 +7,26 @@ namespace Voycar.Api.Web.Entities;
 
 public class User : Entity
 {
-    // Login
+    // Login information
```

Comment:
```suggestion
    // Login Information
```

Reply:
Entweder hier auch das _I_ groß oder in den anderen Dateien das i klein. Ich mache bei den anderen ein Comment hin